client/src/App.vue

```diff
@@ -1,14 +1,14 @@
 <template>
   <div id="app">
     <section class="section">
-      <div class="container">
+      <div class="container is-fluid is-marginless">
```

Comment:
can we remove is-marginless? The page is much easier to read with a bit of a margin.